science/seqan3/Portfile

```diff
@@ -32,8 +32,7 @@ use_xz                  yes
 depends_lib-append      port:bzip2 \
                         port:zlib
 
-patchfiles              patch-spin_delay.hpp-fix-for-Darwin-PPC.diff
```

Comment:
Does this patch not patch anymore?

Reply:
It has been fixed in the updated version of the library (https://github.com/seqan/seqan3/pull/3129/)